cfme/markers/stream_excluder.py

```diff
@@ -34,7 +33,8 @@ def pytest_configure(config):
 
 
 def pytest_itemcollected(item):
-    streams_id = get_streams_id()
+    holder = item.config.pluginmanager.getplugin('appliance-holder')
```

Comment:
**QUESTION:**  Is there a better name than holder for this variable?  I usually expect a variable to express what it is it is holding (hence its always a holder...see the ambiguity?).

Reply:
in the current uses of taking a reference to the appliance holder, im not aware of a nicer spellin thats not also very repetitive (the points towards bad naming of the plug-ins, which is not surprising as its a first iteration

i'd like to abstract going from some-object to appliance a bit (be it pytest items, metafunc, objects that have an actual appliance attribute and so on

Reply:
Here is the conversation we had on irc about this.   The end result is for now we won't change the name of holder, though we both agree it is an awful name:

(09:21:09 AM) oden: This is concerning the holder name.
(09:21:21 AM) oden: So is the appliance holder something different than an appliance?
(09:22:03 AM) ronny: appliance holder is a pytest plugin which manages the appliance objects for a test session as well as worker/master setups with sets of appliances
(09:23:24 AM) oden: It might be better to just call the holder variable appliance_holder.
(09:23:37 AM) oden: calling it just holder feels as meaningful as calling it x
(09:23:42 AM) oden: at least to me.
(09:23:48 AM) ronny: same here
(09:24:00 AM) ronny: that holder objeect is a kind of "x" to me
(09:24:06 AM) oden: right
(09:24:08 AM) oden: hahaha
(09:24:24 AM) oden: So is the holder object going to go away?
(09:24:31 AM) oden: And this is just temporary?
(09:25:02 AM) ronny: i would like to abstract fetching the appliance better since we use it more, and no w that i see us using this in more palces this is a good next step
(09:25:45 AM) oden: Right.   So if you didn't want to change the name now, because something better will replace it soonishly, I'm good with that.
(09:26:11 AM) ronny: but thnaks for being pedantic, i need to add a task for next sprint
(09:26:20 AM) oden: ha